### DISCUSSION TITLE
useState 동작 원리와 클로저 - 사툰사툰 REACT

### DISCUSSION THREAD

**Discussion by giscus** [Wisesaturn/blog#71] [Posts Comments]
# useState 동작 원리와 클로저 - 사툰사툰 REACT

react에선 어떻게 상태를 관리하고 있는지 useState의 구조와 동작 원리를 파악해 보았습니다. | useState 클로저 React 상태 관리 React 18 flushSync 불변성 useState Closure Hooks

https://jaehan.blog/posts/react/useState-%EB%8F%99%EC%9E%91-%EC%9B%90%EB%A6%AC%EC%99%80-%ED%81%B4%EB%A1%9C%EC%A0%80

**Comment by gunwooa** [comment]
잘 읽었습니다!!
